tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/service/grpc/tool/StubGeneratorTestCase.java

```diff
@@ -436,7 +436,7 @@ public void testHelloWorldGatewayWithRepeatedField() throws IllegalAccessExcepti
                 "Expected compilation units not found in compile results.");
         assertEquals(((BLangPackage) compileResult.getAST()).typeDefinitions.size(), 9,
                 "Expected type definitions not found in compile results.");
-        assertEquals(((BLangPackage) compileResult.getAST()).functions.size(), 22,
+        assertEquals(((BLangPackage) compileResult.getAST()).functions.size(), 24,
```

Comment:
I assume this change is due to the new check with method size splitting logic. Did we check the exact reason for this?

Reply:
yes I checked the AST before and after the change. After the change there were more `<init>#` functions

Reply:
ack